src/helpers/graph.js

```diff
@@ -0,0 +1,42 @@
+export function removeNoteFromGraph(nodes, edges, noteToDelete) {
+  let newNodes = [ ...nodes ]
+  let newEdges = [ ...edges ]
+  let parentEdge = newEdges.find(edge => edge.to === noteToDelete.id)

```

Comment:
From `parentEdge` we use only `parentEdge.from`.

Can't we just create a variable
```
const parentId = newEdges.find(edge => edge.to === noteToDelete.id).from;
```
to make the code more readable?